source/fundamentals/indexes.txt

```diff
@@ -371,3 +389,100 @@ The following example creates a unique, descending index on the ``theaterId`` fi
 
 Refer to the :manual:`Unique Indexes page </core/index-unique>` in the MongoDB server manual for more information.
 .. driver-content-end
+
+Remove an Index
+---------------
+
+You can remove any unused index except the default unique index on the
+``_id`` field. 
+
+The following sections show the ways to remove indexes:
+
+- Using an index specification document
+- Using an indexed name field
+- Using a wildcard character to remove all indexes
+
+Remove an Index Using an Index Specification Document
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Pass the **index specification document** to the ``dropIndex()`` method to
+remove an index from a collection. An index specification document is
+a ``Bson`` instance that specifies the type of index you want on the
+specified field. 
+
+The following snippet removes an ascending index on the ``title`` field
+in a collection:
+
+.. code-block:: java
+
+   collection.dropIndex(Indexes.ascending("title"));
+
+.. important::
+
+   If you want to drop a text index, you must use the name of the index
+   instead. See the :ref:`Name Field <name_field>` section for details.
+
+.. _name_field:
+
+Remove an Index Using a Name Field
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Pass the ``name`` field of the index to the ``dropIndex()`` method to
+remove an index from a collection.
+
+If you need to find the name of your index, use the ``listIndexes()``
+method to see the value of the ``name`` fields in your indexes. 
+
+The following snippet retrieves and prints all the indexes in a
+collection: 
+
+.. literalinclude:: /includes/fundamentals/code-snippets/SearchText.java
+   :language: java
+   :dedent:
+   :start-after: begin listIndex
+   :end-before: end listIndex
+
+Your output should resemble something like the following:
```

Comment:
As briefly discussed in our prior Zoom meeting, we should make sure that we're not misleading a user that might not have a collection with a text index. We can frame it in a way that doesn't actually require them to call it on a collection that contains a text index. E.g.

```suggestion
If you call listIndex() on a collection that contains a text index, the output might resemble the following:
```